tools/wurf_options.py

```diff
@@ -63,6 +66,20 @@ def resolve(ctx):
 
 class WurfOptions(Options.OptionsContext):
 
+    def execute(self):
+
+        # Execute the context, this will go out and invoke the option(...)
+        # function defined in the wscript. We have some extra tools that
+        # also define the option(...) function to avoid having to call load
+        # in all wscripts to initialize those, we can do it here.
+
+        self.load('wurf_common_tools')
```

Comment:
What happens if I don't want to load all the common tools in some custom project? (perhaps  a full python project would not need the common tools)